lib/mix/tasks/openapi.spec.json.ex

```diff
@@ -7,6 +7,7 @@ defmodule Mix.Tasks.Openapi.Spec.Json do
 
       $ mix openapi.spec.json --spec PhoenixAppWeb.ApiSpec apispec.json
       $ mix openapi.spec.json --spec PhoenixAppWeb.ApiSpec --pretty=true
+      $ mix openapi.spec.json --spec PhoenixAppWeb.ApiSpec --check=true
```

Comment:
Small stylistic note: Boolean arga can be set to true with just `--arg`, looks a bit nicer than `--arg=true`